docs/multi-echo.rst

```diff
@@ -1,7 +1,11 @@
 Multi-echo fMRI
 ===============
-In multi-echo (ME) fMRI, data are acquired for multiple echo times, resulting in
-multiple time series for each voxel.
+Functional MRI (fMRI) uses the change in blood flow, 
+and specifically blood oxygen level, in the brain to measure brain activity. 
+Only one unit of data, in units of voxel, 
```

Comment:
voxel is not a unit; I don't think this line says what you mean.

Reply:
Possible edit:
> In most echo-planar image (EPI) fMRI sequences, only one brain image is acquired at each repetition time.
> In contrast, in multi-echo (ME) fMRI, data are acquired for multiple echo times, resulting in multiple volumes with varying levels of contrast acquired at each repetition time.

Reply:
> voxel is not a unit; I don't think this line says what you mean.

@jbteves I was under the impression that voxel is a unit of 3D images as pixel is a unit for a 2D images? Could you elaborate a little?

Reply:
> Possible edit:
> 
> > In most echo-planar image (EPI) fMRI sequences, only one brain image is acquired at each repetition time.
> > In contrast, in multi-echo (ME) fMRI, data are acquired for multiple echo times, resulting in multiple volumes with varying levels of contrast acquired at each repetition time.

@tsalo I wanted to mention radio frequency, as it is used in the other sections on that page. I appended it to what you wrote an made another commit, please see if it makes sense!

Reply:
So what I said was wrong, because what I said was overly restrictive for the definition of unit. I think what @tsalo wrote was a better and more helpful suggestion, but I will attempt to unpack what I meant so I don't mislead you. I tend to think of units as things that describe a set quantity, like meters or grams, but a voxel is just something to be counted, which is why my kneejerk reaction was to type "not a unit" (which is wrong). However, the real problem that I failed to identify is that you state that only one unit of data is taken, which indicates that basically you're grabbing one number to represent one quantity at one point in time and space, and then you state that the unit of data is voxel, which is a unit in space only. So those two statements are actually subtly incompatible. What I *should have said* is that it would be more illuminating to say that in most scans, you acquire *one sample* at each point in time and space, whereas in multi-echo you collect *multiple samples* in time and space. Because the voxels are actually basically the same between echoes, it's the samples that are different because each sample has a different intensity. Voxel *is* a unit, but it's not the unit that's pertinent to multi-echo. So I hope that communicates what I originally meant, which was not to tersely state something incorrect. Please let me know if you're more, less, or the same amount of confused. Sorry about all that.

Reply:
@jbteves Yeah I got it! I was saying 

> one "unit" of data, in "units" of voxel

which is quite confusing now that I look back on it too. Looking back at it, it isn't necessary so I replaced it with @tsalo's version.